packages/system/0.5.2/manifest.yml

```diff
@@ -8,7 +8,7 @@ type: integration
 categories:
 - os_system
 - security
-release: beta
+release: experimental
```

Comment:
@exekias @andresrc @fearful-symmetry This is for 7.10 what we have today. Is this change ok? I assume we will release a new version in any case for 7.10?

Reply:
Yes, for 7.10 we will need to release a new version